conan_tests/Dockerfile

```diff
@@ -24,12 +22,14 @@ RUN sudo apt-get update \
     golang \
     pkg-config \
     && sudo rm -rf /var/lib/apt/lists/* \
-    && sudo pip install --upgrade pip \
+    && sudo ln -sf /usr/bin/python3.8 /usr/bin/python \
+    && sudo ln -sf /usr/bin/python3.8 /usr/bin/python3 \
     && wget --no-check-certificate --quiet -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py \
-    && sudo python3 /tmp/get-pip.py \
-    && sudo pip install "virtualenv<20.0.0" \
-    && pip3 install PyGithub \
-    && pip3 install meson
+    && sudo python3.8 /tmp/get-pip.py \
+    && sudo python3.8 -m pip install --upgrade pip \
+    && sudo python3.8 -m pip install "virtualenv<20.0.0" \
+    && sudo python3.8 -m pip install PyGithub \
+    && sudo python3.8 -m pip install meson
```

Comment:
Why installing meson? We are adding meson as a build dependency to all recipes

Reply:
I don't know, it was already there when I arrived. :stuck_out_tongue_closed_eyes: But indeed it doesn't make sense as we have Menson installer.

Reply:
I'll remove it on next commit. Azure test image is broken too. get-pip now uses f-string, which means /usr/bin/python3 must be 3.6 or higher

Reply:
@Croydon now I see: https://github.com/conan-io/conan-docker-tools/pull/175 introduces Meson, but it's for internal tests or to run another stuff internally in CCI. I won't remove now because probably it will affect CCI directly. :sweat: